scripts/mantra_post_test_results

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+if [ -z "$3" ]; then
+    echo "Usage: $0 project-id build-name xml-file"
+    exit 1
+fi
+
+project_id=$1
+
+# TODO: we can add more fields like status or build_url
+request_body=$(cat <<EOF
+{
+  "name": "$2"
+}
+EOF
+)
+
+results_file=$3
+
+builds_endpoint=https://qastatus.mender.io/api/projects/${project_id}/builds
+
+rm -f output
+curl --user ${MANTRA_USERNAME}:${MANTRA_PASSWORD} \
+        -H "Content-Type: application/json" \
+        -d "$request_body" \
+        -o output \
+        "$builds_endpoint"
+
+cat output
```

Comment:
Is this just debugging, or should it be left in?

Reply:
My intention was to leave it.

The script is ignoring errors (no `set -e`) so this is the only way to learn something if the next curl call fails. In any case, there is no sensitive information here and it is very small.

Reply:
How about:
`cat output >&2`